### ISSUE TITLE
´telegram-desktop   5.6.3    6246      latest/stable  telegram-desktop✓  -  Bus-Zugriffsfehler (Speicherabzug geschrieben)

### ISSUE THREAD

**Issue by fizzlifax2** [telegramdesktop/tdesktop#28588]
### Steps to reproduce

Start telegram typing on the shell "telegram-desktop"
Starting by mouseclick over the icon gives no result

### Expected behaviour

Starting of the telegram-desktop

### Actual behaviour

ZugriffsBus-Zugriffsfehler (Speicherabzug geschrieben)
nothing else

### Operating system

Ubuntu 22.04.5 LTS  - 6.8.0-47-generic #47~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Oct  2  GNU/Linux

### Version of Telegram Desktop

 5.6.3    6246 

### Installation source

Snap

**Comment by ilya-fedin** [comment]
Sorry but when trying to reproduce your issue, it was revealed that the behavior matches the expected one. The issue is unreproducible.

**Comment by fizzlifax2** [comment]
I believe You - I tried the following:
I removed telegram from snap on the comandline.
Than I tried to reinstall and got an fatal error what made it impossible to reinstall telegram snap.
Fehler: cannot perform the following tasks:
- Den Konfigurationshook von Snap "telegram-desktop" ausführen, falls vorhanden (run hook "configure": signal: bus error (core dumped))
- I did not insist and installed the pure binary from telegram directly without snap.
This actually works...
I had similar problems with gimp for many months - and fortunately once it worked again...
Thanks a lot for your troubles...
